frontend-react/src/pages/admin/value-set-editor/ValueSetsIndex.tsx

```diff
@@ -19,9 +19,11 @@ import {
     LookupTable,
     LookupTables,
     ValueSet,
-} from "../../../config/endpoints/lookupTables";
+} from "../../../config/endpoints/lookupTables"; // TODO: move to config
+import { MemberType } from "../../../hooks/UseOktaMemberships";
+import { AuthElement } from "../../../components/AuthElement";
 
-const PAGE_TITLE = process.env.REACT_APP_TITLE; // TODO: move to config
```

Comment:
I'd still like to move all of our process.env related globals to a config file

Reply:
Sounds like a good tech-debt item, but out of scope here

Reply:
sure - can you add the comment back then?

Reply:
Didn't even see it had moved. It was next to an import in my branch. 🤷🏼